agent/consul/leader.go

```diff
@@ -183,6 +183,9 @@ func (s *Server) establishLeadership() error {
 
 	s.startAutopilot()
 
+	// Set consistent read readiness state
```

Comment:
You can probably drop the comment here and on line 187 - the function names are clear enough.